src/replica/replica.cpp

```diff
@@ -556,5 +557,17 @@ void replica::init_disk_tag()
     }
 }
 
+error_code replica::store_app_info(app_info &info, const std::string &path)
+{
+    replica_app_info new_info((app_info *)&info);
+    const auto &info_path =
+        path.empty() ? utils::filesystem::path_combine(_dir, ".app-info") : path;
```

Comment:
suggest assign `str` to `const` variable: kAppInfo = ".app-info";

I had added variable: https://github.com/XiaoMi/rdsn/blob/982c2bd180a081fe2c772191cad0ea279e0e2685/src/replica/replica_disk_migrator.cpp#L35

you can refator it into `replica.h`

Reply:
Good idea, code updated.